changelog/druntime.importc-initial-msvc-intrinsics.dd

```diff
@@ -0,0 +1,4 @@
+ImportC, when targeting the Microsoft C runtime, supports a subset of the intrinsics recognised by the MSVC compiler.
```

Comment:
The documentation in the opening comment in the PR is far superior to this readme. Merge it in here.

Reply:
I think I've copied the right part—better now?